pkg/cmd/server/kubernetes/master.go

```diff
@@ -369,6 +353,10 @@ func (c *MasterConfig) RunNodeController() {
 		s.NodeMonitorPeriod.Duration,
 
 		clusterCIDR,
+
+		nil,
```

Comment:
Originally I was unsure whether we wanted this enabled, but since we already have AllocateNodeCIDRs covered it's fine.